sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/ApproximateUnique.java

```diff
@@ -108,6 +120,38 @@
     return new Globally<>(maximumEstimationError);
   }
 
+  /**
+   * Returns a {@link Combine.Globally} that gives a single value that is an estimate of the number
+   * of distinct elements in the input {@code PCollection}.
+   *
+   * @param <T> the type of the elements in the input {@code PCollection}
+   * @param sampleSize the number of entries in the statistical sample; the higher this number, the
+   *     more accurate the estimate will be; should be {@code >= 16}
+   * @param inputCoder the coder for {@code PCollection<T>} where the combineFn will be applied on.
+   * @throws IllegalArgumentException if the {@code sampleSize} argument is too small
+   */
+  public static <T> Combine.Globally<T, Long> combineFn(int sampleSize, Coder<T> inputCoder) {
```

Comment:
The `combineFn()` function here does not actually return the `combineFn` (as you did in `ApproximateQuantile`), please make a change here.